securedrop/tests/client_test.py

```diff
@@ -81,64 +90,65 @@ def assert_initial_registered_users_is_valid(self, ru):
             self.assertTrue(cd.auth.salt)
             self.assertTrue(cd.auth.key)
 
-    def test_aae_initial_json_valid(self):
+
```

Comment:
Remove extra line

Reply:
Gonna let the formater take care of this.